cmd/operator-sdk/bundle/create.go

```diff
@@ -330,3 +342,68 @@ func copyScorecardConfig() error {
 	}
 	return nil
 }
+
+func addSDKMetricLabels() error {
+	var sdkMetricContent string
+
+	content := getSDKMetricsContent()
+
+	for key, value := range content {
+		sdkMetricContent += fmt.Sprintf("LABEL %s=%s\n", key, value)
+	}
+
+	err := projutil.RewriteFileContents(bundle.DockerFile, "LABEL", sdkMetricContent)
+
+	if err != nil {
+		return fmt.Errorf("error rewriting dockerfile with metric labels, %v", err)
+	}
+	return nil
+}
+
+func rewriteBundleImageContents(outputDir, manifestDir string) error {
+	// CopyScorecardConfig to bundle.Dockerfile
+	if err := copyScorecardConfig(); err != nil {
+		return fmt.Errorf("error copying scorecardConfig to bundle image, %v", err)
+	}
+
+	// Add SDK labels to bundle.Dockerfile
+	if err := addSDKMetricLabels(); err != nil {
+		return err
+	}
+
+	// Add sdk metric annotations
+	if err := writeMetricsToAnnotationsYaml(outputDir, manifestDir); err != nil {
+		return fmt.Errorf("error writing sdk metric labels to annotations.yaml, %v", err)
+	}
+
+	return nil
+
+}
+
+// getAnnotationsFilePath return the locations of annotations.yaml. If the user had specified
+// --output dir in the command, then its location would be inside outputDir/metadataDir/. If not
+// the metadataDir is present in the same location as the manifests are present.
+func getAnnotationsFilePath(outputDir, manifestsDir string) string {
+	if outputDir != "" {
+		return filepath.Join(outputDir, bundle.MetadataDir, bundle.AnnotationsFile)
+	}
+	return filepath.Join(filepath.Dir(manifestsDir), bundle.MetadataDir, bundle.AnnotationsFile)
+}
+
+func getSDKMetricsContent() map[string]string {
+	content := projutil.MakeBundleMetricsLabels()
+	return content.Data
+}
```

Comment:
The getSDKMetricsContent() is not required because we can use directly projutil.MakeBundleMetricsLabels()
```suggestion
```